extension-client/src/api/index.ts

```diff
@@ -20,6 +20,11 @@ const schematicsV1 = require('@ibm-cloud/ibm-schematics/dist/schematics/v1');
 import * as auth from '../auth/auth';
 import * as type from '../type/index';
 import * as util from '../util';
+import * as shell from '../command/shell';
+import { posix } from 'path';
+ import * as vscode from 'vscode';
```

Comment:
This needs to be at the top of the other imports

Reply:
We have removed the imports as they were not in use